assets/js/modules/todo-module.js

```diff
@@ -46,6 +36,26 @@ Todo = function () {
 			add_modal: add_modal
 		}
 	}();
+	var table = function() {
+		function generate_table() {
+			//Get data from server using CodeIgniter
+			$.ajax({
+				url: controller+"data_Table",
+				dataType: 'json',
+				success: function() {
```

Comment:
You're not using the data you fetched through ajax. `success: function (response) {
// response is the json from data_Table()
}